packages/atomic/src/components/atomic-no-results-v1/atomic-no-results.tsx

```diff
@@ -0,0 +1,132 @@
+import {
+  buildHistoryManager,
+  HistoryManager,
+  HistoryManagerState,
+  buildSearchStatus,
+  SearchStatus,
+  SearchStatusState,
+  QuerySummary,
+  QuerySummaryState,
+  buildQuerySummary,
+} from '@coveo/headless';
+import {Component, h, Prop, State} from '@stencil/core';
+import {
+  Bindings,
+  BindStateToController,
+  InitializeBindings,
+} from '../../utils/initialization-utils';
+import Lupa from '../../images/lupa-frame.svg';
+
+/**
+ * The `atomic-no-results` component displays search tips and a "Cancel last action" button when there are no results. Any additional content slotted inside of its element will be displayed as well.
+ *
+ * @part cancel-button - The "Cancel last action" button.
+ * @part no-results - The text indicating that no results were found for the search.
+ * @part search-tips - The search tips to help the user correct the query.
+ * @part highlight - The highlighted query.
+ */
+@Component({
+  tag: 'atomic-no-results-v1', //TODO remove v1
+  styleUrl: 'atomic-no-results.pcss',
+  shadow: true,
+})
+export class AtomicNoResults {
+  @InitializeBindings() public bindings!: Bindings;
+  public searchStatus!: SearchStatus;
+  public history!: HistoryManager;
+  public querySummary!: QuerySummary;
+
+  @BindStateToController('searchStatus')
+  @State()
+  private searchStatusState!: SearchStatusState;
+  @BindStateToController('history')
+  @State()
+  private historyState!: HistoryManagerState;
+  @BindStateToController('querySummary')
+  @State()
+  private querySummaryState!: QuerySummaryState;
+  private strings = {
+    searchTips: () => this.bindings.i18n.t('search-tips-v1'),
+    cancelLastAction: () => this.bindings.i18n.t('cancel-last-action'),
+  };
+  @State() public error!: Error;
+
+  /**
+   * Whether to display a button which cancels the last available action.
+   */
+  @Prop() enableCancelLastAction = true;
+
+  public initialize() {
+    this.searchStatus = buildSearchStatus(this.bindings.engine);
+    this.history = buildHistoryManager(this.bindings.engine);
+    this.querySummary = buildQuerySummary(this.bindings.engine);
+  }
+
+  private wrapHighlight(content: string) {
+    return `<span class="font-bold" part="highlight">"${content}"</span>`;
```

Comment:
Could we use the property ::before & ::after to add the quotes? This will allow customer to customize it